Rakefile

```diff
@@ -13,6 +12,24 @@ else
   task :default => :spec
 end
 
+task :appraisal do
+  for_all_gemfiles("exec", "rake")
+end
+
+namespace :appraisal do
+  task :install do
```

Comment:
It might be nice to actually generate a task for each appraisal. 
So they are visible when executed with `rake -T`. Also. this should have `desc` so it appears there with a description.

Reply:
This is what I explain in the second bullet point in https://github.com/justinfrench/formtastic/pull/1304#issuecomment-614248433 that I don't think it's worth it, I believe.

I'll add a `desc`! :+1:

Reply:
Updated!